docs/admin-entity.md

```diff
@@ -0,0 +1,69 @@
+## AdminEntity class
+
+You can register entities with options using the `AdminEntity` class
```

Comment:
Make it clear from the start that there's 2 ways of registering an entity: registering the entity itself (uses all defaults) or an AdminEntity (allows configuration). Then have sections for each of these 2